src/index.js

```diff
@@ -1,4 +1,47 @@
-import{camelCase} from 'lodash';
+const listElement = document.querySelector('ul');
 
+const taskList = [
+  {
+    index: 0,
+    description: 'Going to the gym',
+    completed: false,
+  },
+  {
+    index: 2,
+    description: 'Visit Friends and Family',
+    completed: false,
+  },
+  {
+    index: 1,
+    description: 'Going home',
+    completed: false,
```

Comment:
- [ ] Kindly render the tasks according to the `index` values in the array. That means that _**Going to the gym**_ should be the first, _**Going home**_ should be the second and _**Visit Friends and Family**_ should be the third based on the values of the `index`.